rasa_sdk/forms.py

```diff
@@ -656,29 +555,259 @@ async def run(
     def __str__(self) -> Text:
         return f"FormAction('{self.name()}')"
 
-    def _get_entity_type_of_slot_to_fill(
+
+class SeparateValidationAction(Action, ABC):
```

Comment:
can we keep the old name / create an alias with a deprecation so it doesn't break for users when updating?